pi5/board.go

```diff
@@ -390,7 +265,7 @@ type pinctrlpi5 struct {
 	dtBaseNodePath string    // file path referring to base of device tree: /proc/device-tree
 	gpioNodePath   string    // file path referring to gpio chip's location within the device-tree. retrieved from 'aliases' node: /proc/device-tree/axi/pcie@12000/rp1/gpiochip0
 	virtAddr       *byte     // base address of mapped virtual page referencing the gpio chip data
-	physAddr       uint64    // base addres of the gpio chip data in dev/mem/
+	physAddr       uint64    // base address of the gpio chip data in dev/mem/
```

Comment:
:+1: Thanks for fixing this!

Are you sure about how you've formatted the last word? I had expected `/dev/mem` (indicating it's a global path and `mem` is a device and not a directory), rather than `dev/mem` (suggesting that `dev/` is a subdirectory within the current location, and `mem/` is another directory within it).

Reply:
ill fix that too thanks for calling it out